plugins/copr.py

```diff
@@ -240,9 +267,11 @@ def _need_root(cls):
 
     @classmethod
     def _guess_chroot(cls):
-        """ Guess which choot is equivalent to this machine """
+        """ Guess which chroot is equivalent to this machine """
         # FIXME Copr should generate non-specific arch repo
-        dist = platform.linux_distribution()
+        dist = CHROOT_CONFIG
```

Comment:
I do not like the usage of global variable. Can you make CHROOT_CONFIG class attribute and refer to it as self.cli.CHROOT_CONFIG (and rename cls to self)?